lib/yamlcpp/CMakeLists.txt

```diff
@@ -93,7 +93,7 @@ endif()
 
 target_compile_options(yaml-cpp
   PRIVATE
-    $<${not-msvc}:-Wall -Wextra -Wshadow -Weffc++ -Wno-long-long>
```

Comment:
Need to be careful here, because this will get erased on the next drop. For libswoc I have a script that does these kind of tweaks, we might want to have one in the ATS code base for yaml-cpp (and perhaps move the libswoc one here).

Reply:
That's ok.  yamlcpp has this flag but then has lots of shadowed variables.  Its just really noisy so I removed it so I could see other issues.